Use the latest version of the NHS.UK frontend library

AdamChrimes opened the pull request:
- Use the latest version of the NHS.UK frontend library ([v3.0.4](https://github.com/nhsuk/nhsuk-frontend/blob/master/CHANGELOG.md#304---24-march-2020))
- Add logo and open graph assets

mikemonteith commented:
What are the logo and open graph images for?
I can't see that they're used anywhere.

AdamChrimes commented:
@mikemonteith I added them mainly for the open graph assets to be used, but needs abit more work so I'll pick that up elsewhere.

I've removed those assets from this PR
